android/sdk/src/main/java/io/o2mc/sdk/Datastreams/DatastreamsDispatcher.java

```diff
@@ -40,14 +42,17 @@ public void update(Activity activity, JSONObject generalInfo) {
 
     /**
      * Used to set dispatchThreshold, when x datacontainers have been gathered send data to endpoint
-     * @param threshold
+     *
+     * @param threshold int - default 5.
      */
     public void setDispatchThreshold(int threshold) {
         this.dispatchThreshold = threshold;
     }
 
+    @SuppressWarnings("unchecked")
+    // Ignore because there's an if statement which does check the amount of items to serialize
     public void dispatch(DataContainer dataContainer) {
-        dataContainers.add(dataContainer.asJson());
+                dataContainers.add(dataContainer.asJson());
```

Comment:
weird indentation

Reply:
granted

though most of these indentation 'issues' are a result of merging through the web interface. in an IDE I'd always press my shortcut for formatting the entire file, so I don't care much about indentation issues on github because I'm sure it'll be fixed in the next commit by habit.

Reply:
valid point nonetheless :smile: